### PR TITLE
feat：Post action 中支持获取父任务ID #10968

### PR DESCRIPTION
**Pull request by Kzhou991**

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=11001) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-ci?pullRequest=11001) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-ci?pullRequest=11001) it.</sub>

File: src/backend/ci/core/store/api-store/src/main/kotlin/com/tencent/devops/store/pojo/common/StoreConstants.kt [hunk 1]
```diff
@@ -58,6 +58,8 @@ const val CLOSE = "close" // 开关关闭
 const val ATOM_POST = "post" // 插件post
 const val ATOM_POST_FLAG = "postFlag" // 插件post标识
 const val ATOM_POST_ENTRY_PARAM = "postEntryParam" // 插件post操作入口参数
+const val ATOM_POST_PARENT_TASK_ID = "postActionParentTaskId" // 插件后置动作父任务id
```
Comment (carlyin0801) [inline src/backend/ci/core/store/api-store/src/main/kotlin/com/tencent/devops/store/pojo/common/StoreConstants.kt:61]: postActionParentTaskId换成parentTaskId

File: src/backend/ci/core/worker/worker-common/src/main/kotlin/com/tencent/devops/worker/common/task/market/MarketAtomTask.kt [hunk 2]
```diff
@@ -256,7 +259,7 @@ open class MarketAtomTask : ITask() {
             // 开关关闭则不再写入插件私有配置到input.json中
             inputVariables.putAll(getAtomSensitiveConfMap(atomCode))
         }
-        writeInputFile(atomTmpSpace, inputVariables)
+//        writeInputFile(atomTmpSpace, inputVariables)
```
Comment (carlyin0801) [inline src/backend/ci/core/worker/worker-common/src/main/kotlin/com/tencent/devops/worker/common/task/market/MarketAtomTask.kt:262]: 没用的注释去掉

File: src/backend/ci/core/worker/worker-common/src/main/kotlin/com/tencent/devops/worker/common/task/market/MarketAtomTask.kt [hunk 3]
```diff
@@ -151,6 +153,7 @@ open class MarketAtomTask : ITask() {
         val map = JsonUtil.toMutableMap(data)
         // 该插件执行的工作空间绝对路径
         val workspacePath = workspace.absolutePath
+        val additionalOptions = taskParams["additionalOptions"]
```
Comment (carlyin0801) [inline src/backend/ci/core/worker/worker-common/src/main/kotlin/com/tencent/devops/worker/common/task/market/MarketAtomTask.kt:156]: 这个变量定义这么靠前干什么，另外"additionalOptions"换成Element::additionalOptions.name更加优雅

**Pull request closed without merging**